1.0/index.md

```diff
@@ -16,7 +16,7 @@ title: Welcome
       </a>
       <a href="https://github.com/polymer">
         <paper-button class="github" unresolved>
-          <core-icon icon="social:post-github"></core-icon> View on GitHub
+          <core-icon src="components/component-download-button/icons/github.png"></core-icon> View on GitHub
```

Comment:
Im a little hesitant to rely on a component's image for the site. Can we just check in the log and reference it from static/images?